docs/hacking-guide/en/history.md

```diff
@@ -0,0 +1,19 @@
+History
+=======
+
+Artemis started with a code donation grant by RedHat in Oct 2014. During this process we took a snapshot of codebase and included it as an initial commit on the apache repository.
```

Comment:
Perhaps reword this slightly.

The Apache ActiveMQ Artemis project was started in October 2014.  The Artemis code base was seeded with a code donation, granted by Red Hat, of the HornetQ project.  The code donation process consisted of taking a snapshot of the latest HornetQ code base and contributing this snapshot as an initial git commit into the Artemis git repository. 

The HornetQ commit history is preserved, and can be accessed here: <>

Credit should be given to those developers who contributed to the HornetQ project.  The top 10 committers are highlighted here:  <>

For more information please visit the [HornetQ GitHub project].